sklearn/exceptions.py

```diff
@@ -154,3 +155,12 @@ class UndefinedMetricWarning(UserWarning):
     .. versionchanged:: 0.18
        Moved from sklearn.base.
     """
+
+
+class SklearnDeprecationWarning(DeprecationWarning):
```

Comment:
For the record, ScrapyDeprecationWarning derives from Warning and not DeprecationWarning ;-): https://github.com/scrapy/scrapy/blob/129421c7e31b89b9b0f9c5f7d8ae59e47df36091/scrapy/exceptions.py#L48.